src/ts/components/subtitleslistbox.ts

```diff
@@ -0,0 +1,55 @@
+import {ListBox} from './listbox';
+import {ListSelectorConfig} from './listselector';
+import {UIInstanceManager} from '../uimanager';
+
+/**
+ * A element that is similar to a select box where the user can select a subtitle
+ */
+export class SubtitlesListBox extends ListBox {
```

Comment:
The name should be in singular, i.e. `SubtitleListBox`, Analogous to `SubtitleSelectBox` (filename too)